src/diffusers/models/attention.py

```diff
@@ -113,15 +114,24 @@ def __init__(
         double_self_attention: bool = False,
         upcast_attention: bool = False,
         norm_elementwise_affine: bool = True,
+        norm_eps: int = 1e-5,
         norm_type: str = "layer_norm",
         final_dropout: bool = False,
         attention_type: str = "default",
+        ada_norm_continous_conditioning_embedding_dim: Optional[int] = None,
+        ada_norm_bias: Optional[int] = None,
+        ff_inner_dim: Optional[int] = None,
+        ff_bias: bool = True,
+        attention_out_bias: bool = True,
+        skip_ff: bool = False,
```

Comment:
Do we skip the feed-forward in MUSE?

Reply:
We skip the feed-forward in the attention blocks in the input and output resblocks but not in the actual transformer -- i.e.

input res block with attn -> transformer blocks -> output res block with attn
        ^                                                                                 ^
        |                                                                                  |
   skips ff                                                                          skips ff

Reply:
Hmmm can we maybe just directly code the attention layers inside the res block then and not use ` BasicTransformerBlock`? Think this class is being customized too much for MUSE here. Let's just move the attn layers and norms directly into `OpenMuseResBlock` so that we don't need to customized it that much here?